src/engine-scripts/puppet/collapsedTocState.js

```diff
@@ -23,6 +23,6 @@ module.exports = async ( page, hashtags ) => {
 		btn.click();
 	}, collapseTocButton );
 
-	const tocButtonSelector = isStickyHeader ? '#vector-sticky-header-toc-checkbox,#p-sticky-header-toc-checkbox' : '#vector-toc-collapsed-button';
+	const tocButtonSelector = isStickyHeader ? '#vector-sticky-header-toc-label' : '#vector-page-titlebar-toc-label';
```

Comment:
Given that https://pixel.wmcloud.org/ checks the latest release branch, can this be changed to be backwards compatible?:

```
	const tocButtonSelector = isStickyHeader ? '#vector-sticky-header-toc-label,#vector-sticky-header-toc-checkbox' : '#vector-page-titlebar-toc-label,#vector-toc-collapsed-button';
```

Reply:
...apparently there's a bug in example line as I don't see the TOC appear in the test cases. I guess if it's too much effort we can just wait for the branch cut tomorrow :)